internal/scorecard/plugins/olm_tests.go

```diff
@@ -379,14 +437,27 @@ func getUsedResources(proxyPod *v1.Pod) ([]schema.GroupVersionKind, error) {
 }
 
 // Run - implements Test interface
-func (t *AnnotationsContainExamplesTest) Run(ctx context.Context) *schelpers.TestResult {
+func (t *CSVValidationTest) Run(ctx context.Context) *schelpers.TestResult {
 	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
-	if t.CSV.Annotations != nil && t.CSV.Annotations["alm-examples"] != "" {
-		res.EarnedPoints = 1
-	}
-	if res.EarnedPoints == 0 {
-		res.Suggestions = append(res.Suggestions, fmt.Sprintf("Add an alm-examples annotation to your CSV to pass the %s test", t.GetName()))
+	res.EarnedPoints = 1
+
+	csvValidator := validation.ClusterServiceVersionValidator
+	results := csvValidator.Validate(t.CSV)
+	for _, r := range results {
+		if r.HasError() {
+			res.EarnedPoints = 0
+			for _, e := range r.Errors {
+				res.Errors = append(res.Errors, e)
+			}
+		}
+		if r.HasWarn() {
+			res.EarnedPoints = 0
```

Comment:
Should the number of earned points be 0 here? Warnings should still pass this test right?

Reply:
agree, I've changed it to let warnings pass the test.